missing/kqueue_inotify.c

```diff
@@ -64,9 +64,12 @@ fs_sysctl(const int name) {
 		file = fopen("/proc/sys/fs/inotify/max_user_watches", "r");
 
 		if (file == NULL || fgets(line, sizeof(line), file) == NULL)
-		    err(1, "max_user_watches");
-		value = atoi(line);
-		fclose(file);
+			warnx("failed to read max_user_watches");
```

Comment:
As an Android user, would this warning become tiring? At first it's useful to see what going on, but we can't fix this and we'll do something reasonable instead.

Perhaps just set `value = 0` here and add a comment about why this might happen?

Reply:
Good point, warning has been changed to a comment